OpenEXR/IlmImf/ImfOutputFile.cpp

```diff
@@ -255,7 +255,7 @@ writeLineOffsets (OPENEXR_IMF_INTERNAL_NAMESPACE::OStream &os, const vector<Int6
 {
     Int64 pos = os.tellp();
 
-    if (pos == -1)
+    if (pos == Int64(-1))
```

Comment:
static_cast<Int64>

Reply:
I've converted all constructor casts to static_cast